image.go

```diff
@@ -310,6 +310,13 @@ func (img *imageHDU) Write(data interface{}) error {
 		for _, v := range data {
 			w.writeI16(v)
 		}
+	case []uint16:
```

Comment:
we should probably add tests for these new cases in `TestImageRW` (in `image_test.go`)

Reply:
added